RoddyCore/src/de/dkfz/roddy/core/ProjectLoader.groovy

```diff
@@ -314,6 +315,19 @@ class ProjectLoader {
     }
 
     /**
+     * If --additionalImports is used, try to attach all configuration objects to the currently loaded
```

Comment:
I don't understand the comment. Is that not something that should happen anyway unconditionally?